### ISSUE TITLE
Deletion of a field will cause creation time to be changed

### ISSUE THREAD

**Issue by yuzijie** [Jermolene/TiddlyWiki5#2579]
Hello,
If you add a field to a tiddler, only the _modified_ field will change, however, if you **remove** a field from a tiddler, the _created_ field will also be updated to the current time.

Please check for this bug. Thanks.

**Comment by twMat** [comment]
Confirmed (by, on *tw.com* tiddler *HelloThere*, simply removing the *list* field).

Good catch @yuzijie !

**Comment by yuzijie** [comment]
Hi @twMat @pmario @tobibeer @Jermolene 

I raised this issue about 2 weeks ago, since it has not attracted too much people’s attention, I decided to do some investigation by myself, then I found this [commit](https://github.com/Jermolene/TiddlyWiki5/commit/eb9fb6635db7b7b1a50f8acd2712ddfdda9fc340) and this [post](https://groups.google.com/forum/#!msg/tiddlywiki/wxdAoLW62Gk/T_4KBB2NAQAJ).

It seems that the behavior of updating _created_ field after the deletion of other fields is added intentionally with the purpose of covering some edge cases.

However, logically speaking, deletion of a field does not imply that the old tiddler is reborn. And what my biggest concern is that this behavior will cause _created_ field to be **unreliable**.

Imagine if somebody has a list that sorted by _created_ field, and one day, the owner removed a field from a tiddler, then, the order of that list will also be changed even without being noticed by the owner.

I wish that this issue can be fixed as soon as possible, because creation date is an important property, and should not be changed so easily.
